templates/nextcloud-migrate.nomad

```diff
@@ -50,8 +52,8 @@ job "nextcloud-migrate" {
         {{- with secret "liquid/nextcloud/nextcloud.maria" }}
           MYSQL_PASSWORD = {{.Data.secret_key | toJSON }}
         {{- end }}
-        {{- with secret "liquid/nextcloud/nextcloud.admin" }}
-          OC_PASS = {{.Data.secret_key | toJSON }}
+        {{- with secret "liquid/nextcloud/nextcloud.uploads" }}
+          UPLOADS_USER_PASS = {{.Data.secret_key | toJSON }}
```

Comment:
Since the other one is called `NEXTCLOUD_ADMIN_PASSWORD`, could this one be `UPLOADS_USER_PASSWORD`?